src/Groups/julia_GAP.jl

```diff
@@ -0,0 +1,270 @@
+import Hecke: base_ring, dim, elements, ncols, nrows, order
+import GAP: FFE
+
+
+# TODO: at this moment, only finite fields are implemented; in future we want to use infinite fields
+const TempMatType = fq_nmod_mat
+
+function FieldGapToHecke(F::GapObj)
```

Comment:
I don't think this file belong into this pull request...

Reply:
I added this file to test direct products of matrix groups. Should I remove it to avoid confusion?